com.beavernotes.beavernotes.yaml

```diff
@@ -0,0 +1,55 @@
+app-id: com.beavernotes.beavernotes
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+command: beaver-notes
+finish-args:
+  - --device=dri
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+  - --filesystem=home
+  - --filesystem=/mnt
+  - --filesystem=/run/media
+  - --filesystem=/media
```

Comment:
This is no different than using `--filesystem=host`. Instead of manually putting the combinations, you can just use that.

But why do you need this? Electron supports filechooser portals

Reply:
Thanks for the suggestion. I'll swap "the" with --filesystem=host. I know Electron supports a file chooser; however, in order to perform synchronization across devices, the app needs to be able to access the path given by the user, which can be a folder in the home, on a server, or a cloud provider.